addons/udes_stock/models/stock_picking_type.py

```diff
@@ -395,6 +395,42 @@ class StockPickingType(models.Model):
         " (this only applies if 'Check Package Type' is enabled)",
     )
 
+    # Package count options
+    u_show_package_count = fields.Boolean(
+        string="Show Package Count",
+        default=False,
+        help="Enables Package count button on kanban view",
+    )
+    u_package_count_location_id = fields.Many2one(
```

Comment:
Have we considered using the default source location of the picking type instead of adding a new field?

Reply:
I think we probably could for this use case, although might we want to option to specify a different location in the future?

Reply:
In that case, we probably want multiple locations, since a picking type can be used from different source locations.
I would keep it simple for now, and iterate later.

Reply:
Thank you guys for your review and suggestions. 
I have remove the pack_count_location_id field and now using default_location_src_id